lib/Conf.ml

```diff
@@ -563,7 +568,10 @@ module Formatting = struct
         , `Hexadecimal
         , "$(b,hexadecimal) mode escapes every character." ) ]
     in
-    C.choice ~names ~all ~doc ~section
+    let deprecated =
+      C.deprecated ~since_version:"0.14.0" "There is no equivalent."
```

Comment:
Can we have a more precise behavior where the warning (warning only, not the --help manual) depends on the selected value? Like:
- `preserve` : this option is not required anymore, character escaping is now preserved by default, 
- others : this feature has been removed, character escaping is now preserved by default.

Reply:
I'm not sure if this is necessary.
The option was already not required when set to the default value, is this something we should warn regardless if the option is deprecated ?
The default didn't change because of the deprecation and we can't predict if it will change after we removed the option.

Reply:
It make sense. Alright let's merge if it's ready :)